designer/server/src/test/resources/config/business-cases/simple-streaming-use-case-designer.conf

```diff
@@ -8,105 +8,17 @@ authentication:  {
 baseModelConfig {
   classPath: [
     "engine/flink/management/dev-model/target/scala-"${scala.major.version}"/devModel.jar",
+    "engine/flink/components/base-unbounded/target/scala-"${scala.major.version}"/flinkBaseUnbounded.jar",
```

Comment:
Can you replace `devModel.jar` with:
```
"defaultModel/target/scala-"${scala.major.version}"/defaultModel.jar"
```
?

Reply:
Done
- It changed the available SPEL utils, I changed the helper used in tests
- I had to add additionally flinkBase.jar to have "dead-end" sink

Reply:
Nice, thanks, on step closer to untangle this mess around dev model